### PR TITLE
docs(universal): getting started, serving with Cloud Functions, and prerendering

### PR DESCRIPTION
**Pull request by jamesdaniels**
### Checklist

   - Issue number for this PR: #nnn (required)
   - Docs included?: (yes/no; required for all API/functional changes) 
   - Test units included?: (yes/no; required) 
   - In a clean directory, `yarn install`, `yarn test` run successfully? (yes/no; required)

### Description

Extracted code from [jamesdaniels/AngularFireDocs](https://github.com/jamesdaniels/AngularFireDocs).

### CODE REVIEW

**Comment by jamesdaniels** [comment]
FYI @codediodeio this is where I think I'm going with the Universal Docs, thoughts?

File: README.md [hunk 1]
```diff
@@ -76,7 +76,7 @@ Firebase offers two cloud-based, client-accessible database solutions that suppo
 
 #### Realtime Database
 
-> `AngularFireDatabase` allows you to work with the Realtime Database, Firebase's original database. It's an efficient, low-latency solution for mobile apps that require synced states across clients in realtime.
+> `AngularFireDatabase` allows you to work with the Realtime Database, Firebase's original database. It's an efficient, low-latency solution for mobile apps that require synced states across clients in real-time.
```
Comment (davideast) [inline README.md]: "realtime" over "real-time" this goes way back to the early Firebase days.

**Comment by jamesdaniels** [comment]
@davideast addressed and flushed out more.

File: docs/universal/getting-started.md [hunk 2]
```diff
@@ -1,11 +1,11 @@
-# Server-side Rendering with Universal
+# Getting started with Angularfire and Universal
```
Comment (davideast) [inline docs/universal/getting-started.md]: mega nit: AngularFire

File: docs/universal/getting-started.md [hunk 3]
```diff
@@ -77,31 +74,44 @@ app.get('*', (req, res) => {
...
-Create a new file named `webpack.server.config.js` to bundle the express app from previous step. 
+Create a new file named `webpack.server.config.js` to bundle the express app from previous step.
 
 
 ```js
 const path = require('path');
 const webpack = require('webpack');
 
-const APP_NAME = 'YOUR_PROJECT_NAME';
+const APP_NAME = 'YOUR_PROJECT_NAME'; // TODO: replace me!
```
Comment (davideast) [inline docs/universal/getting-started.md]: I think we need more context around `'YOUR_APP_NAME`. Something comment like "This is the same as the folder name...".

Comment (jamesdaniels) [reply docs/universal/getting-started.md]: Better yet, my mid-term plan here is to read the paths from the `angular.json` file; need to fiddle there. SGTY?

**Pull request merged**